wave/resources/views/checkout.blade.php

```diff
@@ -1,11 +1,24 @@
-<script src="https://cdn.paddle.com/paddle/paddle.js"></script>
+<script src="https://cdn.paddle.com/paddle/v2/paddle.js"></script>
```

Comment:
```suggestion
<script src="https://cdn.paddle.com/paddle/v2/paddle.js" defer></script>
```

Reply:
This does not seem to be working unfortunately:

```
Uncaught ReferenceError: Paddle is not defined
    at pricing:984:5
```

Reply:
Yes, I guess so. I didn't manage to combine my suggestion in a single entry. You need to defer the script after this one too (https://github.com/thedevdojo/wave/pull/100#discussion_r1544362982).
We need to make sure that we try to instantiate Paddle after the paddle script is loaded.